ocsci/main.py

```diff
@@ -45,50 +43,19 @@ def init_ocsci_conf(arguments=[], default_config=OCSCI_DEFAULT_CONFIG):
     """
     custom_config = get_param('--ocsci-conf', arguments)
     cluster_config = get_param('--cluster-conf', arguments)
-    config_data = ocsci.config.to_dict()
     with open(default_config) as file_stream:
         default_config_data = yaml.safe_load(file_stream)
-    update_dict_recursively(config_data, default_config_data)
+        rendered_defaults = render_yaml_with_j2_context(default_config_data)
```

Comment:
Before the rendering of jinja2 was done at the end once all configs were merged, so you could use j2 variables in all config files. But now you are rendering via jinja just default_config_data and the rest of the config files loosing ability to use jinja2 there. So if we don't want to add additional logic, it should be removed from documentation and from examples of config where I mentioned those functionality.

Reply:
To me, it seems unnecessarily complex to allow jinja2 in configuration. If a lot of people disagree, then we should discuss it.

It's definitely on my to-do list to submit docs updates with this PR

Reply:
Yes I agree that we can simplify this and avoid using jinja templates in the rest of config files, just pointing that it needs to be documented. Cause currently it's documented in examples like you can use those jinja templates.